src/vmm/Cargo.toml

```diff
@@ -29,6 +29,8 @@ snapshot = { path = "../snapshot"}
 utils = { path = "../utils" }
 virtio_gen = { path = "../virtio_gen" }
 vm-memory = { path = "../vm-memory" }
+vm-allocator = "0.1.0"
+
```

Comment:
This should be added to line 20, in order to keep separation between local crates and remote ones. Also remove the extra line.